<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Rajesh" failures="0" tests="4" name="com.leaftaps.testcases.TC001_VerifyLogin" time="37.086" errors="1" timestamp="2025-04-10T18:01:05 IST" skipped="1">
  <testcase classname="com.leaftaps.testcases.TC001_VerifyLogin" name="runLogin1" time="0.554"/>
  <testcase classname="com.leaftaps.testcases.TC001_VerifyLogin" name="runLogin" time="23.789"/>
  <testcase classname="com.leaftaps.testcases.TC001_VerifyLogin" name="preCondition" time="12.743">
    <error message="Test failed: See report for details." type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException: Test failed: See report for details.
at com.framework.utils.Reporter.reportStep(Reporter.java:110)
at com.framework.utils.Reporter.reportStep(Reporter.java:128)
at com.framework.selenium.api.base.SeleniumBase.startApp(SeleniumBase.java:643)
at com.framework.testng.api.base.ProjectSpecificMethods.preCondition(ProjectSpecificMethods.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:833)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:600)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
]]>
    </error>
  </testcase> <!-- preCondition -->
  <testcase classname="com.leaftaps.testcases.TC001_VerifyLogin" name="runLogin2" time="0.000">
    <skipped/>
  </testcase> <!-- runLogin2 -->
</testsuite> <!-- com.leaftaps.testcases.TC001_VerifyLogin -->
